modules/cpmso-openstack-without-az-config.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-openstack-ts-az-config_{context}"]
+= Configuring {rh-openstack} clusters that have machines with root volume availability zones after an upgrade
+
+For some clusters that run on {rh-openstack-first} that you upgrade, you must manually update machine resources before you can use control plane machine sets if the following configurations are true:
+
+* You upgraded the cluster from {product-title} 4.13 to 4.14.
+ 
+* The cluster infrastructure is installer-provisioned.
+
+* Machines were configured to use availability zones.
```

Comment:
Same comment as https://github.com/openshift/openshift-docs/pull/66212/files#r1361012607

Reply:
@EmilienM This link isn't resolving to a particular file for me. Taking a guess, is it something like this?

```suggestion
* Machines were distributed across multiple availability zones.
```

Reply:
yes, exactly